safe-global/web-core
# feat: allow initialisation of SDK with unsupported mastercopies

## What it solves

SDK initialisation on unsupported networks.

## How this PR fixes it

If the mastercopy is an official deployment that is not supported on the current network, e.g. L1 contract on L2 network, we now initialise the SDK.

We now fetch the Safe version directly from the contract (as it is `null` from backend) and detect whether the address matches an official Safe deployment.

## How to test it

1. Open an unsupported Safe and observe that it is now possible to execute a transaction.
2. Observe the warning in the transaction modal.

It is possible to create an "unsupported" either via the CLI or by using the Copycat app and deploying a mainnet address to an L2 network, e.g. Polygon.

## Screenshots

![image](https://user-images.githubusercontent.com/20442784/224648606-b54f5fc6-1163-43d5-82c5-6d3564f7061c.png)

## Checklist
